template/src/main.js

```diff
@@ -3,7 +3,7 @@
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
```

Comment:
Should change the file name to main.ts?

Reply:
It is changed via metalsmith - that was the basic idea of the PR. 
See my initial comment and `meta.js`